oonib/report/handlers.py

```diff
@@ -75,13 +88,13 @@ def get_report_path(report_id):
 
         report_filename = get_report_path(self.report_id)
         try:
-            with open(report_filename) as fd:
-                g = yaml.safe_load_all(fd)
-                report_details = g.next()
+            open(report_filename).close()
         except IOError:
             raise e.ReportNotFound
 
-        dst_filename = report_file_name(self.archive_dir, report_details)
+        dst_filename = report_file_name(archive_dir=self.archive_dir,
+                                        report_details=self.report_details,
+                                        report_id=self.report_id)
```

Comment:
I find it confusing that there is a variable named `report_filename` and a function named `report_file_name`